box-content-sample/src/main/java/com/box/androidsdk/sample/MainActivity.java

```diff
@@ -59,10 +61,10 @@ protected void onCreate(Bundle savedInstanceState) {
         mAdapter = new BoxItemAdapter(this);
         mListView.setAdapter(mAdapter);
         BoxConfig.IS_LOG_ENABLED = true;
-        BoxConfig.CLIENT_ID = "<YOUR_CLIENT_ID>";
-        BoxConfig.CLIENT_SECRET = "<YOUR_CLIENT_SECRET>";
+        BoxConfig.CLIENT_ID = "m9bcgls0kffgyclmt9hz0jbs9ua7m0yy";
```

Comment:
Take care not to include client id/secrets in commits. Because this is now in your git history -- you will likely have to squash your commits into one and submit a new pull request. Note: This does not currently apply to the preview sdk as that is still private.